displays/cavity_display/backend/backend_cavity.py

```diff
@@ -154,14 +158,44 @@ def get_fault_counts(
         """
         result: DefaultDict[str, FaultCounter] = defaultdict(FaultCounter)
 
-        for fault in self.faults.values():
-            result[fault.tlc] = max(
-                result[fault.tlc],
-                fault.get_fault_count_over_time_range(
-                    start_time=start_time, end_time=end_time
-                ),
-            )
+        data = get_values_over_time_range(
```

Comment:
Can I request a type annotation for data? I think it would be `data: Dict[str, ArchiveDataHandler]`